atlasdb-tests-shared/src/main/java/com/palantir/atlasdb/keyvalue/impl/AbstractKeyValueServiceTest.java

```diff
@@ -1079,6 +1079,23 @@ private void testGetRangeWithTimestamps(boolean reverse) {
     }
 
     @Test
+    public void testGetRangeOfTimestampsReturnsAllRows() {
+        keyValueService.put(TEST_TABLE,
+            ImmutableMap.of(
+                Cell.create(row0, column0), value0_t0,
+                Cell.create(row1, column0), value0_t0,
+                Cell.create(row2, column0), value0_t0),
+            TEST_TIMESTAMP);
+        RangeRequest range = RangeRequest.all().withBatchHint(1);
+        List<RowResult<Set<Long>>> results = ImmutableList.copyOf(
+                keyValueService.getRangeOfTimestamps(TEST_TABLE, range, TEST_TIMESTAMP + 1));
+        assertEquals(3, results.size());
+        assertTrue(Arrays.equals(row0, results.get(0).getRowName()));
+        assertTrue(Arrays.equals(row1, results.get(1).getRowName()));
+        assertTrue(Arrays.equals(row2, results.get(2).getRowName()));
```

Comment:
For these, I'd prefer we use `assertArrayEquals(expected, actual)` instead. We'll get a more helpful assertion failure message if something breaks here.